src/usb/uf2/uf2cfg.h

```diff
@@ -1,9 +1,15 @@
 #include "boards.h"
 #include "dfu_types.h"
 
-// Family ID for updating Application
+// Legacy Family ID for updating Application
```

Comment:
it is not legacy, it is generic one, uf2 with 
- vid/pid can be upgraded
- family =  generic ada52840 should also be accepted as well.

Reply:
Yes, the code allows both.

Reply:
I mean you need to remove the `legacy` in the comment